package veil.oo.test;

public class BadStuffHappened extends Exception {

    public BadStuffHappened(Throwable cause) {
        super(cause);
    }

}
